.travis.yml

```diff
@@ -28,17 +32,19 @@ install:
   # Useful for debugging any issues with conda
   - conda info -a
   # Create conda test environment
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy pandas xarray requests beautifulsoup4 lxml netCDF4 h5py pytest-cov pytest-ordering coveralls future
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
   - conda activate test-environment
-  # Dependencies not available through conda, install through pip
-  - pip install pytest-flake8
+  # Once pysat 3.0 is released, remove the custom pysat install and change
+  # this to:
+  # - while read requirement; do conda install --yes $requirement; done < requirements.txt
+  - conda install --yes h5py numpy pandas xarray
```

Comment:
```suggestion
  - conda install --yes h5py numpy pandas xarray portalocker
```

Reply:
This isn't needed because a few lines down we go and get that from the pysat requirements file.